### PR TITLE
Coverage update

### PR DESCRIPTION
**Pull request by snelis**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -14,6 +14,3 @@ not_skip = __init__.py
 
 [wheel]
 universal = 1
-
-[coverage:run]
-omit = python_project/tests/*
```
Comment (jeduden) [inline setup.cfg]: is that path correct ?

Comment (snelis) [inline setup.cfg]: its a removal of the line

Comment (jeduden) [inline setup.cfg]: good point ;)

**Comment by jeduden** [comment]
LGTM

**Pull request merged**